docs/docs-next/docs/guides/data-assets/passing-data-between-assets.md

```diff
@@ -1,7 +1,124 @@
 ---
-title: "Passing data between assets"
-sidebar_position: 3
+title: How to Pass Data Between Assets
+description: Learn how to pass data between assets in Dagster
+sidebar_position: 30
 sidebar_label: "Passing data between assets"
+last_update: 
+    date: 2024-08-11
+    author: Pedram Navid
 ---
 
-# Passing data between assets
\ No newline at end of file
+As you develop your data pipeline, you'll likely need to pass data between assets. By the end of this guide, you'll have a solid understanding of the different approaches to passing data between assets and when to use each one.
+
+---
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- A basic understanding of Dagster concepts such as assets and resources
+- Dagster installed, as well as the `dagster-duckdb-pandas` package
+</details>
+
+---
+
+## Overview
+
+In Dagster, assets are the building blocks of your data pipeline and it's common to want to pass data between them. This guide will help you understand how to pass data between assets. 
+
+There are three ways of passing data between assets:
```

Comment:
I think it would be useful to include a fourth "fake" case which is "you do not pass data between assets _because your pipeline is not processing data directly_" 

This would be something like:

```python 

@asset 
def people(): 
 """ call the lambda function that loads people"""
 
@asset 
def birds(): 
 """ call the lambda function that loads birds"""
 
 
 @asset(
   deps = [people, birds]
 ) 
def people_and_birds(): 
 """ call the stored procedure that concats people and birds"""
 ```